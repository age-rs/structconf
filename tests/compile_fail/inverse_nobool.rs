//! An inverted argument doesn't make sense for something that doesn't
//! implement `std::ops::Not`.

use structconf::StructConf;

#[derive(StructConf)]
struct Config {
    #[conf(negated_arg)]
    pub value: f64,
}

fn main() {}
