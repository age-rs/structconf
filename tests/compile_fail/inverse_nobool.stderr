error: `negated_arg` is incompatible with `field's type`.
 --> $DIR/inverse_nobool.rs:9:9
  |
9 |     pub value: f64,
  |         ^^^^^
